packages/terra-action-footer/src/CenteredActionFooter.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ActionFooterContainer from './_ActionFooterContainer';
+import styles from './CenteredActionFooter.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Actions to be displayed in the center socket
+   */
+  center: PropTypes.node,
```

Comment:
Should this just be children?

Reply:
We had the discussion about it, I think that we decided to go with `center` as a means of keeping the API consistent with using `start` and `end` in the normal `ActionFooter` component, but I could be wrong

Reply:
There wasn't necessarily a consensus, but the discussion happened here: https://github.com/cerner/terra-core/issues/1286#issuecomment-370476119